tools/eclipse-temurin/19-jre-jammy/Dockerfile

```
@@ -0,0 +1,2 @@
+FROM --platform=$BUILDPLATFORM eclipse-temurin:19-jre-jammy
+RUN apk upgrade --no-cache && rm -rf /var/cache/apk/*
```

`apk` is for Alpine, for Ubuntu you need `apt` or `apt-get`:
```suggestion
RUN apt-get update && apt-get upgrade -y && apt-get clean && rm -rf /var/lib/apt/lists/*
```

---

Ah, good catch. That's a side effect of me trying an Alpine image before